### PR TITLE
[CI:DOCS]Do not run compose tests with CI:DOCS

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -448,6 +448,7 @@ apiv2_test_task:
 compose_test_task:
     name: "compose test on $DISTRO_NV"
     alias: compose_test
+    only_if: &not_docs $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*'
```
Comment (cevich) [inline .cirrus.yml:451]: This should just be simply `only_if: *not_docs`

Comment (baude) [reply .cirrus.yml:451]: ty!

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**